test/unit/mdc-textfield/mdc-textfield.test.js

```diff
@@ -192,6 +234,14 @@ test('#adapter.registerInputBlurHandler adds a "blur" event handler on the input
   td.verify(handler(td.matchers.anything()));
 });
 
+test('#adapter.registerTextFieldClickHandler adds a "click" event handler on the input element', () => {
```

Comment:
Related to a comment I had above, we should have a test for the reverse of this, for deregister.